robottelo/hosts.py

```diff
@@ -1102,6 +1102,24 @@ def install_tracer(self):
             raise ContentHostError('There was an error installing katello-host-tools-tracer')
         self.execute('katello-tracer-upload')
 
+    def register_to_cdn(self):
+        """Subscribe satellite to CDN"""
+        self.remove_katello_ca()
+        major_version = self.os_version.major
+        release_version = f'{major_version}Server' if major_version < 8 else f'{major_version}'
```

Comment:
This all can go into `register_contenthost` method which is more related to registeration.

Reply:
I think not, this option is valid only for CDN registration and not for dogfood, and if we do that we will be hardcoding `releasever` param for each registration, which will be for contenthosts as well